kotest-core/src/commonMain/kotlin/io/kotest/core/spec/configuration.kt

```diff
@@ -47,6 +47,12 @@ abstract class TestConfiguration {
     */
    var defaultTestConfig: TestCaseConfig? = null
 
+   /**
+    * Returns the number of concurrent root test cases of Spec that can be executed.
+    * Defaults to 1.
+    */
+   var threadsForSpec: Int = 1
```

Comment:
I would prefer this to be called `threads` and the doc should read

"Returns the size of the thread pool that is used to execute root tests in this spec.
Note: This only applies to the JVM platform and has no effect on other platforms."

Because we may decide later to allow concurrency even with a single thread, for instance if you are testing coroutine code, you don't necessarily need > 1 thread.

Reply:
renaming - done.
Docs is not my area. Sorry for English.

Reply:
On my opinion `threads` for spec and `threads` for test will confuse users
```kotlin
class Test : FunSpec({
   
   threads = 3

   test("a").config(threads = 2) {
   }
}
```
First `threads` like default config for Spec class and second `threads` like config just for this test case
And users may forget about 
```kotlin
defaultTestConfig = TestCaseConfig(threads = 3)
```
are different configuration parameters